pages/authLogin/authLogin.ejs

```diff
@@ -104,6 +127,18 @@
             </a>
           <% } %>
         </div>
+        <% if (institutionAuthnProviders.length > 0) { %>
+          <div class="institution-header text-muted my-3">
+            Institution sign-on
+          </div>
+          <div class="login-methods">
+          <% institutionAuthnProviders.forEach((provider) => { %>
+            <a href="<%= provider.url %>" class="btn btn-outline-dark btn-block">
+              <%= provider.name %>
+            </a>
+          <% }) %>
+          </div>
+        <% } %>
```

Comment:
Could we add a second copy of the most-recently-used institution (stored in a cookie) in a separate area at the top? That is, I'd like to get an obvious "Sign in with Illinois" button when I hit this page (as well as having Illinois being listed in it's usual order in case I read down the list).

Reply:
Would you be willing to pull this out into a separate ticket + PR?

Reply:
Yep! https://github.com/PrairieLearn/PrairieLearn/issues/6137

Reply:
Thanks! Going to mark this resolved; we can pick that one up later.